components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41Utils.java

```diff
@@ -2781,7 +2782,7 @@ public static void addTenant(org.apache.stratos.common.beans.TenantInfoBean tena
         try {
             TenantMgtUtil.addClaimsToUserStoreManager(tenant);
         } catch (Exception e) {
-            String msg = "Error in granting permissions for tenant " + tenantDomain;
+            String msg = "Error in granting permissions for tenant " + tenantDomain + e.getLocalizedMessage();
```

Comment:
No need of using "e.getLocalizedMessage()" as we do not support localization IMO.